INFO  started: Metals version 0.8.3 in workspace '/home/pi/workspace/api-gateway'
INFO  time: initialize in 6.44s
INFO  running '/usr/lib/jvm/java-11-openjdk-armhf/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /tmp/metals11357252209210158463/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
INFO  [info] Loading settings for project api-gateway-build from plugins.sbt,metals.sbt ...
INFO  [info] Loading project definition from /home/pi/workspace/api-gateway/project
INFO  [warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
INFO  [info] Loading settings for project root from build.sbt,docker-opt.sbt,version.sbt,release.sbt ...
INFO  [info] Loading settings for project kafka from build.sbt ...
INFO  [info] Loading settings for project etcdDiscovery from build.sbt ...
ERROR /home/pi/workspace/api-gateway/base-models/build.sbt:3: warning: Adaptation of argument list by inserting () is deprecated: this is unlikely to be what you want.
ERROR         signature: DefinableTask.:=(v: S): sbt.Def.Setting[sbt.Task[S]]
ERROR   given arguments: <none>
ERROR  after adaptation: DefinableTask.:=((): Unit)
ERROR publish := ()
ERROR         ^
INFO  [info] Loading settings for project base-models from build.sbt ...
INFO  [info] Set current project to api-gateway (in build file:/home/pi/workspace/api-gateway/)
INFO  [success] Generated .bloop/kafka-test.json
INFO  [success] Generated .bloop/kafka.json
INFO  [error] lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.3/kanela-agent-1.0.3-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.3/kanela-agent-1.0.3-sources.jar
INFO  [error] https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6-sources.jar
INFO  [error] https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.11/scala-library-2.12.11-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.11/scala-library-2.12.11-sources.jar
INFO  [error] 
INFO  [error] 	at lmcoursier.internal.shaded.coursier.Artifacts$.$anonfun$fetchArtifacts$14(Artifacts.scala:302)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
INFO  [error] 	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: lmcoursier.internal.shaded.coursier.cache.ArtifactError$DownloadError: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.3/kanela-agent-1.0.3-sources.jar
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:1017)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: java.net.ConnectException: Connection timed out (Connection timed out)
INFO  [error] 	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
INFO  [error] 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
INFO  [error] 	at java.base/java.net.Socket.connect(Socket.java:609)
INFO  [error] 	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:285)
INFO  [error] 	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
INFO  [error] 	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:182)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1071)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1069)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1068)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1587)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1507)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1505)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1504)
INFO  [error] 	at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:334)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirectTo(CacheUrl.scala:152)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirect(CacheUrl.scala:162)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:304)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:277)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$33(FileCache.scala:347)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.$anonfun$downloading$1(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withUrlLock(CacheLocks.scala:91)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/com/pauldijou/jwt-json-common_2.12/4.3.0/jwt-json-common_2.12-4.3.0.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/com/pauldijou/jwt-json-common_2.12/4.3.0/jwt-json-common_2.12-4.3.0.jar
INFO  [error] https://repo1.maven.org/maven2/dev/zio/izumi-reflect_2.12/0.12.0-M0/izumi-reflect_2.12-0.12.0-M0.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/dev/zio/izumi-reflect_2.12/0.12.0-M0/izumi-reflect_2.12-0.12.0-M0.jar
INFO  [error] https://repo1.maven.org/maven2/dev/zio/zio_2.12/1.0.0-RC18-2/zio_2.12-1.0.0-RC18-2.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/dev/zio/zio_2.12/1.0.0-RC18-2/zio_2.12-1.0.0-RC18-2.jar
INFO  [error] 
INFO  [error] 	at lmcoursier.internal.shaded.coursier.Artifacts$.$anonfun$fetchArtifacts$14(Artifacts.scala:302)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
INFO  [error] 	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: lmcoursier.internal.shaded.coursier.cache.ArtifactError$DownloadError: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/com/pauldijou/jwt-json-common_2.12/4.3.0/jwt-json-common_2.12-4.3.0.jar
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:1017)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: java.net.ConnectException: Connection timed out (Connection timed out)
INFO  [error] 	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
INFO  [error] 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
INFO  [error] 	at java.base/java.net.Socket.connect(Socket.java:609)
INFO  [error] 	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:285)
INFO  [error] 	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
INFO  [error] 	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:182)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1071)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1069)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1068)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1587)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1507)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1505)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1504)
INFO  [error] 	at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:334)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirectTo(CacheUrl.scala:152)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirect(CacheUrl.scala:162)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:304)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:277)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$33(FileCache.scala:347)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.$anonfun$downloading$1(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withUrlLock(CacheLocks.scala:91)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3-sources.jar
INFO  [error] 
INFO  [error] 	at lmcoursier.internal.shaded.coursier.Artifacts$.$anonfun$fetchArtifacts$14(Artifacts.scala:302)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
INFO  [error] 	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: lmcoursier.internal.shaded.coursier.cache.ArtifactError$DownloadError: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3-sources.jar
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:1017)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] Caused by: java.net.ConnectException: Connection timed out (Connection timed out)
INFO  [error] 	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
INFO  [error] 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
INFO  [error] 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
INFO  [error] 	at java.base/java.net.Socket.connect(Socket.java:609)
INFO  [error] 	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:285)
INFO  [error] 	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
INFO  [error] 	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:182)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
INFO  [error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1071)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1069)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1068)
INFO  [error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1587)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1507)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1505)
INFO  [error] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
INFO  [error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795)
INFO  [error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1504)
INFO  [error] 	at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)
INFO  [error] 	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:334)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirectTo(CacheUrl.scala:152)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirect(CacheUrl.scala:162)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:304)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:277)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$33(FileCache.scala:347)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.$anonfun$downloading$1(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withUrlLock(CacheLocks.scala:91)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:996)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1029)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:318)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:505)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:53)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:76)
INFO  [error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:506)
INFO  [error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
INFO  [error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
INFO  [error] 	at scala.util.Success.map(Try.scala:213)
INFO  [error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
INFO  [error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
INFO  [error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
INFO  [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
INFO  [error] 	at java.base/java.lang.Thread.run(Thread.java:834)
INFO  [error] (base-models / updateClassifiers) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.3/kanela-agent-1.0.3-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.3/kanela-agent-1.0.3-sources.jar
INFO  [error] https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6-sources.jar
INFO  [error] https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.11/scala-library-2.12.11-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.11/scala-library-2.12.11-sources.jar
INFO  [error] (update) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/com/pauldijou/jwt-json-common_2.12/4.3.0/jwt-json-common_2.12-4.3.0.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/com/pauldijou/jwt-json-common_2.12/4.3.0/jwt-json-common_2.12-4.3.0.jar
INFO  [error] https://repo1.maven.org/maven2/dev/zio/izumi-reflect_2.12/0.12.0-M0/izumi-reflect_2.12-0.12.0-M0.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/dev/zio/izumi-reflect_2.12/0.12.0-M0/izumi-reflect_2.12-0.12.0-M0.jar
INFO  [error] https://repo1.maven.org/maven2/dev/zio/zio_2.12/1.0.0-RC18-2/zio_2.12-1.0.0-RC18-2.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/dev/zio/zio_2.12/1.0.0-RC18-2/zio_2.12-1.0.0-RC18-2.jar
INFO  [error] (etcdDiscovery / updateClassifiers) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
INFO  [error] https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3-sources.jar: download error: Caught java.net.ConnectException: Connection timed out (Connection timed out) (Connection timed out (Connection timed out)) while downloading https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3-sources.jar
INFO  [error] Total time: 261 s (04:21), completed 6 Apr 2020, 23:29:20
INFO  build tool exit: 1
INFO  time: ran 'sbt bloopInstall' in 6m29s
ERROR sbt command failed: /usr/lib/jvm/java-11-openjdk-armhf/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /tmp/metals11357252209210158463/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol local --socket /tmp/bsp-launcher7212701464957755319/bsp.socket'...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
No server running at 127.0.0.1:8212, let's fire one...
Resolving ch.epfl.scala:bloop-frontend_2.12:1.4.0-RC1-105-118a551b...
Starting bloop server at 127.0.0.1:8212...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
[0m[32m[D][0m Loading 2 projects from '/home/pi/workspace/api-gateway/.bloop'...
[0m[32m[D][0m Loading project from '/home/pi/workspace/api-gateway/.bloop/kafka-test.json'
[0m[32m[D][0m Loading project from '/home/pi/workspace/api-gateway/.bloop/kafka.json'
[0m[32m[D][0m Cache miss for scala instance org.scala-lang:scala-compiler:2.12.11.
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/jline/jline/2.14.6/jline-2.14.6.jar
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/org/fusesource/jansi/jansi/1.12/jansi-1.12.jar
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6.jar
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-compiler/2.12.11/scala-compiler-2.12.11.jar
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.11/scala-library-2.12.11.jar
[0m[32m[D][0m   => /home/pi/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.12.11/scala-reflect-2.12.11.jar
[0m[32m[D][0m Missing analysis file for project 'kafka'
[0m[32m[D][0m Missing analysis file for project 'kafka-test'
[0m[32m[D][0m Waiting for a connection at local:///tmp/bsp-launcher7212701464957755319/bsp.socket...
The server is listening for incoming connections at local:///tmp/bsp-launcher7212701464957755319/bsp.socket...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /home/pi/.cache/metals/bsp.trace.json
ERROR Timeout waiting for 'build/initialize' response
ERROR Failed to connect with build server, no functionality will work.
java.util.concurrent.TimeoutException
	at java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)
	at scala.meta.internal.metals.BuildServerConnection$.initialize(BuildServerConnection.scala:259)
	at scala.meta.internal.metals.BuildServerConnection$.$anonfun$fromSockets$1(BuildServerConnection.scala:203)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:834)

INFO  started: Metals version 0.8.4 in workspace '/home/pi/workspace/api-gateway'
INFO  time: initialize in 6.5s
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol local --socket /tmp/bsp-launcher1667093469784058295/bsp.socket'...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
WARN  no build target for: /home/pi/workspace/api-gateway/app/controllers/RegistrationController.scala
Waiting for the bsp connection to come up...
INFO  running '/usr/lib/jvm/java-11-openjdk-armhf/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /tmp/metals5270080841954062236/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
No server running at 127.0.0.1:8212, let's fire one...
Resolving ch.epfl.scala:bloop-frontend_2.12:1.4.0-RC1-190-ef7d8dba...
Starting bloop server at 127.0.0.1:8212...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
Attempting a connection to the server...
[0m[32m[D][0m Waiting for a connection at local:///tmp/bsp-launcher1667093469784058295/bsp.socket...
The server is listening for incoming connections at local:///tmp/bsp-launcher1667093469784058295/bsp.socket...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /home/pi/.cache/metals/bsp.trace.json
ERROR Timeout waiting for 'build/initialize' response
ERROR Failed to connect with build server, no functionality will work.
java.util.concurrent.TimeoutException
	at java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)
	at scala.meta.internal.metals.BuildServerConnection$.initialize(BuildServerConnection.scala:259)
	at scala.meta.internal.metals.BuildServerConnection$.$anonfun$fromSockets$1(BuildServerConnection.scala:203)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:834)

INFO  [info] Loading settings for project api-gateway-build from plugins.sbt,metals.sbt ...
INFO  [info] Loading project definition from /home/pi/workspace/api-gateway/project
INFO  [warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
INFO  [info] Loading settings for project root from build.sbt,version.sbt,release.sbt ...
INFO  [info] Loading settings for project kafka from build.sbt ...
INFO  [info] Loading settings for project etcdDiscovery from build.sbt ...
ERROR /home/pi/workspace/api-gateway/base-models/build.sbt:3: warning: Adaptation of argument list by inserting () is deprecated: this is unlikely to be what you want.
ERROR         signature: DefinableTask.:=(v: S): sbt.Def.Setting[sbt.Task[S]]
ERROR   given arguments: <none>
ERROR  after adaptation: DefinableTask.:=((): Unit)
ERROR publish := ()
ERROR         ^
INFO  [info] Loading settings for project base-models from build.sbt ...
INFO  [info] Set current project to api-gateway (in build file:/home/pi/workspace/api-gateway/)
INFO  [success] Generated .bloop/base-models-test.json
INFO  [success] Generated .bloop/base-models.json
INFO  [success] Generated .bloop/kafka.json
INFO  [success] Generated .bloop/kafka-test.json
INFO  [success] Generated .bloop/etcdDiscovery.json
INFO  [success] Generated .bloop/etcdDiscovery-test.json
INFO  [warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
INFO  [warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
INFO  [success] Generated .bloop/root-test.json
INFO  [success] Generated .bloop/root.json
INFO  [success] Total time: 42 s, completed Apr 11, 2020, 9:59:28 AM
INFO  build tool exit: 0
INFO  time: ran 'sbt bloopInstall' in 2m46s
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol local --socket /tmp/bsp-launcher7846623371293596732/bsp.socket'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Waiting for a connection at local:///tmp/bsp-launcher7846623371293596732/bsp.socket...
The server is listening for incoming connections at local:///tmp/bsp-launcher7846623371293596732/bsp.socket...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /home/pi/.cache/metals/bsp.trace.json
INFO  time: connected to build server in 0.31s
INFO  Connected to Build server v1.4.0-RC1-190-ef7d8dba
INFO  time: imported build in 0.71s
WARN  Could not find java sources in None. Java symbols will not be available.
INFO  time: indexed workspace in 50s
INFO  compiling base-models (7 scala sources)
INFO  Non-compiled module 'compiler-bridge_2.12' for Scala 2.12.11. Compiling...
INFO    Compilation completed in 59.412s.
INFO  compiling etcddiscovery (2 scala sources)
INFO  time: compiled base-models in 1m14s
WARN  there was one deprecation warning (since 2.6.0); re-run with -deprecation for details
INFO  time: compiled etcdDiscovery in 11s
INFO  compiling etcddiscovery-test (1 scala source)
INFO  compiling root (86 scala sources and 1 java source)
INFO  time: compiled etcdDiscovery-test in 13s
INFO  compiling root-test (6 scala sources)
INFO  time: compiled root in 2m12s
INFO  time: compiled root-test in 7.8s
Apr 11, 2020 10:07:49 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.InterruptedException
java.util.concurrent.CompletionException: java.lang.InterruptedException
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:36)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.InterruptedException
	at scala.meta.internal.metals.FutureCancelToken.checkCanceled(FutureCancelToken.scala:27)
	at scala.meta.pc.OffsetParams.checkCanceled(OffsetParams.java:29)
	at scala.meta.internal.metals.CompilerOffsetParams.checkCanceled(CompilerOffsetParams.scala:7)
	at scala.meta.internal.pc.CompletionProvider.$anonfun$completions$1(CompletionProvider.scala:73)
	at scala.collection.Iterator$$anon$10.next(Iterator.scala:461)
	at scala.collection.Iterator.toStream(Iterator.scala:1417)
	at scala.collection.Iterator.toStream$(Iterator.scala:1416)
	at scala.collection.AbstractIterator.toStream(Iterator.scala:1431)
	at scala.collection.Iterator.$anonfun$toStream$1(Iterator.scala:1417)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1173)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1163)
	at scala.collection.immutable.StreamIterator.$anonfun$next$1(Stream.scala:1061)
	at scala.collection.immutable.StreamIterator$LazyCell.v$lzycompute(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator$LazyCell.v(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator.hasNext(Stream.scala:1055)
	at scala.collection.convert.Wrappers$IteratorWrapper.hasNext(Wrappers.scala:31)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:134)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:40)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.write(TypeAdapterRuntimeTypeWrapper.java:69)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.write(ReflectiveTypeAdapterFactory.java:125)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.write(ReflectiveTypeAdapterFactory.java:243)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:423)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:55)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at com.google.gson.Gson.toJson(Gson.java:648)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:145)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:140)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:59)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.lambda$handleRequest$1(RemoteEndpoint.java:281)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714)
	... 8 more

Apr 11, 2020 10:08:17 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.InterruptedException
java.util.concurrent.CompletionException: java.lang.InterruptedException
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:36)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.InterruptedException
	at scala.meta.internal.metals.FutureCancelToken.checkCanceled(FutureCancelToken.scala:27)
	at scala.meta.pc.OffsetParams.checkCanceled(OffsetParams.java:29)
	at scala.meta.internal.metals.CompilerOffsetParams.checkCanceled(CompilerOffsetParams.scala:7)
	at scala.meta.internal.pc.CompletionProvider.$anonfun$completions$1(CompletionProvider.scala:73)
	at scala.collection.Iterator$$anon$10.next(Iterator.scala:461)
	at scala.collection.Iterator.toStream(Iterator.scala:1417)
	at scala.collection.Iterator.toStream$(Iterator.scala:1416)
	at scala.collection.AbstractIterator.toStream(Iterator.scala:1431)
	at scala.collection.Iterator.$anonfun$toStream$1(Iterator.scala:1417)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1173)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1163)
	at scala.collection.immutable.StreamIterator.$anonfun$next$1(Stream.scala:1061)
	at scala.collection.immutable.StreamIterator$LazyCell.v$lzycompute(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator$LazyCell.v(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator.hasNext(Stream.scala:1055)
	at scala.collection.convert.Wrappers$IteratorWrapper.hasNext(Wrappers.scala:31)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:134)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:40)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.write(TypeAdapterRuntimeTypeWrapper.java:69)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.write(ReflectiveTypeAdapterFactory.java:125)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.write(ReflectiveTypeAdapterFactory.java:243)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:423)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:55)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at com.google.gson.Gson.toJson(Gson.java:648)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:145)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:140)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:59)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.lambda$handleRequest$1(RemoteEndpoint.java:281)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714)
	... 8 more

INFO  compiling root (1 scala source)
INFO  time: compiled root in 1.62s
INFO  compiling root (1 scala source)
INFO  time: compiled root in 1.49s
INFO  compiling root (1 scala source)
INFO  time: compiled root in 2.02s
Apr 11, 2020 10:09:30 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.InterruptedException
java.util.concurrent.CompletionException: java.lang.InterruptedException
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:36)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.InterruptedException
	at scala.meta.internal.metals.FutureCancelToken.checkCanceled(FutureCancelToken.scala:27)
	at scala.meta.pc.OffsetParams.checkCanceled(OffsetParams.java:29)
	at scala.meta.internal.metals.CompilerOffsetParams.checkCanceled(CompilerOffsetParams.scala:7)
	at scala.meta.internal.pc.CompletionProvider.$anonfun$completions$1(CompletionProvider.scala:73)
	at scala.collection.Iterator$$anon$10.next(Iterator.scala:461)
	at scala.collection.Iterator.toStream(Iterator.scala:1417)
	at scala.collection.Iterator.toStream$(Iterator.scala:1416)
	at scala.collection.AbstractIterator.toStream(Iterator.scala:1431)
	at scala.collection.Iterator.$anonfun$toStream$1(Iterator.scala:1417)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1173)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1163)
	at scala.collection.immutable.StreamIterator.$anonfun$next$1(Stream.scala:1061)
	at scala.collection.immutable.StreamIterator$LazyCell.v$lzycompute(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator$LazyCell.v(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator.hasNext(Stream.scala:1055)
	at scala.collection.convert.Wrappers$IteratorWrapper.hasNext(Wrappers.scala:31)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:134)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:40)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.write(TypeAdapterRuntimeTypeWrapper.java:69)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.write(ReflectiveTypeAdapterFactory.java:125)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.write(ReflectiveTypeAdapterFactory.java:243)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:423)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:55)
	at com.google.gson.Gson.toJson(Gson.java:669)
	at com.google.gson.Gson.toJson(Gson.java:648)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:145)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:140)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:59)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.lambda$handleRequest$1(RemoteEndpoint.java:281)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714)
	... 8 more

INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.45s
INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.15s
INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.69s
INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.53s
INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.25s
INFO  compiling root (2 scala sources)
INFO  time: compiled root in 1.42s
Apr 11, 2020 10:13:29 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 34
No more data in the server stdin, exiting...
Unexpected exception when forwarding server stdin ---> client stdout
No more data in the server stdin, exiting...
java.io.IOException: com.sun.jna.LastErrorException: [104] Connection reset by peer
	at bloop.sockets.UnixDomainSocket$UnixDomainSocketInputStream.doRead(UnixDomainSocket.java:171)
	at bloop.sockets.UnixDomainSocket$UnixDomainSocketInputStream.read(UnixDomainSocket.java:156)
	at java.base/java.nio.channels.Channels$ReadableByteChannelImpl.read(Channels.java:388)
	at bloop.launcher.bsp.BspBridge.copyContents(BspBridge.scala:289)
	at bloop.launcher.bsp.BspBridge.forwardStreamContents(BspBridge.scala:275)
	at bloop.launcher.bsp.BspBridge.$anonfun$wireBspConnectionStreams$2(BspBridge.scala:228)
	at bloop.bloopgun.core.Shell$$anon$2.run(Shell.scala:155)
Caused by: com.sun.jna.LastErrorException: [104] Connection reset by peer
	at bloop.sockets.UnixDomainSocketLibrary.read(Native Method)
	at bloop.sockets.UnixDomainSocket$UnixDomainSocketInputStream.doRead(UnixDomainSocket.java:169)
	... 6 more
ERROR timeout: build server 'bloop' during shutdown
